packages/api/lambdas/bulk-operation.js

```diff
@@ -137,6 +138,7 @@ async function bulkGranuleDelete(payload) {
  *
  * @param {Object} payload
  * @param {string} payload.workflowName - name of the workflow that will be applied to each granule.
+ * @param {Object} [payload.meta] = optional meta to add to workflow input
```

Comment:
```suggestion
 * @param {Object} [payload.meta] - optional meta to add to workflow input
```

Reply:
er, and queueName should probably have optional if we're doing both [] and 'optional'